app/views/publishers/vacancies/review.html.slim

```diff
@@ -34,7 +34,7 @@
       p.govuk-body = t("jobs.preview_listing.message")
       = govuk_button_link_to t("buttons.preview_job_listing"), organisation_job_preview_path(@vacancy.id), class: "preview-listing-gtm", id: "vacancy-review-preview"
 
-      h3.govuk-heading-m = t("jobs.submit_listing.heading", count: @vacancy.publish_on.today? ? 0 : 1, date: @vacancy.publish_on.to_s.strip)
+      h3.govuk-heading-m = t("jobs.submit_listing.heading", count: @vacancy.publish_on&.today? ? 0 : 1, date: @vacancy.publish_on.to_s.strip)
```

Comment:
there is a second call to that method that needs the safe operator:

`h3.govuk-heading-m = t("jobs.submit_listing.heading", count: @vacancy.publish_on&.today? ? 0 : 1, date: @vacancy.publish_on&.to_s&.strip)`

Reply:
It's actually fine - nil.to_s.strip is ""